app/Http/Controllers/UnIgnoreProjectController.php

```diff
@@ -10,6 +10,12 @@ public function __invoke(Project $project)
     {
         $project->update(['ignored' => false]);
 
+				if (request()->wantsJson()) {
+					return response()->json([
+						'success' => true
+					], 200);
```

Comment:
Instead of returning `'success' => true`, let's just return the positive status code.

`response()->json([], 200)` should do it I think.